juju/juju
# Remove loggo v1 from a direct dependency

The loggo v1 package was being directly imported instead of the v2 version. Swapping the import url to v2 fixes this. One less direct dependency allows us to understand what we own and need to maintain.

This was spotted when interning the `juju/charm` package.



## Checklist




## QA steps

Ensure that debug-log still works correctly.

```sh
$ juju bootstrap lxd test
$ juju add-model default
$ juju deploy ubuntu
$ juju debug-log -m controller
$ juju debug-log
```

## Links

**Jira card:** JUJU-
